src/main/java/com/google/devtools/build/lib/skyframe/SingleToolchainResolutionFunction.java

```diff
@@ -210,41 +205,50 @@ private static boolean checkConstraints(
       @Nullable EventHandler eventHandler,
       ConstraintCollection toolchainConstraints,
       String platformType,
-      PlatformInfo platform) {
+      PlatformInfo platform,
+      Label toolchainTypeLabel, Label toolchainLabel) {
```

Comment:
Please put toolchainLabel on a new line for consistency,

Reply:
Done.